### DISCUSSION TITLE
Create repository with "who has the link" option!

### DISCUSSION THREAD

**Discussion by BENZOOgataga** [community/community#37742] [Repositories]
Hey GitHub! Just here to feedback something i've seen on YouTube playlists, it'd be great if we could create repository with the "who has the repo link can see it" option, I'll explain if this isn't clear. Like in YouTube playlists, you are able to create it with three options: public (everyone can see it), not listed (only people who has the link can see it) and private (only you can see it). What I would like is to be able to do this but on GitHub!

**Comment by jfullstackdev** [comment]
you have that feature on Gist, 

![image](https://user-images.githubusercontent.com/47092464/198875941-53e36089-c924-4208-b412-965063677f4c.png)

only those who has the link will be able to open it but it's highly discouraged to send confidential info,

if you want it in a regular repo, make the repo private and invite collaborators,
